client/src/main/scala/drt/client/components/ToolTips.scala

```diff
@@ -0,0 +1,65 @@
+package drt.client.components
+
+import drt.shared.Queues.{EGate, Queue}
+import japgolly.scalajs.react.vdom.TagOf
+import japgolly.scalajs.react.vdom.html_<^._
+import org.scalajs.dom.html
+import org.scalajs.dom.html.Span
+
+object ToolTips {
+
+  def depBanksOrDesksTip(queue: Queue) = if (queue == EGate) depBanksTooltip else depDesksTooltip
```

Comment:
I'm wondering how best to arrange these tooltips as we have quite a few in here and it could start to get difficult to find what we're looking for.

A couple of ideas
1) arrange in alphabetical order
2) group into common subjects or pages

Reply:
yeah, seems reasonable, I wasn't sure whether to just inline them, but it does kind of make sense to have them all in one place for future amendments